mrtrix3_connectome/execute.py

```diff
@@ -76,15 +76,17 @@ def execute(): #pylint: disable=unused-variable
 
     if app.ARGS.analysis_level == 'preproc':
 
-        preproc_shared = PreprocShared(app.ARGS.gdc)
+        preproc_shared = PreprocShared(app.ARGS.gdc,
+                                       app.ARGS.concat_denoise,
+                                       app.ARGS.eddy_cubicflm,
+                                       app.ARGS.eddy_mbs)
 
         for session_to_process in sessions_to_analyze:
             app.console(f'Commencing execution for session: {"_".join(session_to_process)}')
             run_preproc(app.ARGS.bids_dir,
                         session_to_process,
                         preproc_shared,
                         app.ARGS.t1w_preproc,
-                        app.ARGS.concat_denoise,
```

Comment:
Tried running mrtrix3_connectome 'preproc' using this branch but am getting a TypeError due to a missing parameter when it goes to run preproc.run.run_preproc(). The run_preproc() is still expecting the concat_denoise argument but that has been removed here. Looks to still be used in the function (line 363).